From: Guido Guenther <agx@sigxcpu.org>
Date: Thu, 26 Jun 2008 20:01:38 +0200
Subject: Allow libvirt group to access the socket
Forwarded: no

This is the group based access to libvirt functions as it was used
in Ubuntu for quite long.

Debian uses root + policykit for the same. But since Ubuntu did it
the group based way for so long people are used to that, so we keep it.

There are some related tests (if augeas is enabled as build depend) that need
to be adapted in their expected output, that is done in:
 d/p/ubuntu/daemon-augeas-fix-expected.patch


Index: libvirt/daemon/libvirtd.conf
===================================================================
--- libvirt.orig/daemon/libvirtd.conf	2016-07-13 16:12:30.306986666 +0200
+++ libvirt/daemon/libvirtd.conf	2016-07-13 16:12:30.146986668 +0200
@@ -82,14 +82,14 @@
 # without becoming root.
 #
 # This is restricted to 'root' by default.
-#unix_sock_group = "libvirt"
+unix_sock_group = "libvirt"
 
 # Set the UNIX socket permissions for the R/O socket. This is used
 # for monitoring VM status only
 #
 # Default allows any user. If setting group ownership, you may want to
 # restrict this too.
-#unix_sock_ro_perms = "0777"
+unix_sock_ro_perms = "0777"
 
 # Set the UNIX socket permissions for the R/W socket. This is used
 # for full management of VMs
@@ -99,7 +99,7 @@
 #
 # If not using PolicyKit and setting group ownership for access
 # control, then you may want to relax this too.
-#unix_sock_rw_perms = "0770"
+unix_sock_rw_perms = "0770"
 
 # Set the UNIX socket permissions for the admin interface socket.
 #
@@ -138,7 +138,7 @@
 #
 # To restrict monitoring of domains you may wish to enable
 # an authentication mechanism here
-#auth_unix_ro = "none"
+auth_unix_ro = "none"
 
 # Set an authentication scheme for UNIX read-write sockets
 # By default socket permissions only allow root. If PolicyKit
@@ -147,7 +147,7 @@
 #
 # If the unix_sock_rw_perms are changed you may wish to enable
 # an authentication mechanism here
-#auth_unix_rw = "none"
+auth_unix_rw = "none"
 
 # Change the authentication scheme for TCP sockets.
 #
